frontend/src/app/components/docs/api-docs-data.ts

```diff
@@ -0,0 +1,4821 @@
+const bitcoinNetworks = ["", "testnet", "signet"];
+const liquidNetworks = ["liquid", "liquidtestnet"];
+
+export const wsApiDocsData = {
+  codeTemplate: {
+    curl: `/api/v1/ws`,
+    commonJS: `
+  const { %{0}: { websocket } } = mempoolJS();
+
+  const ws = websocket.initClient({
+    options: ['blocks', 'stats', 'mempool-blocks', 'live-2h-chart'],
+  });
+
+  ws.addEventListener('message', function incoming({data}) {
+    const res = JSON.parse(data.toString());
+    if (res.block) {
+      document.getElementById("result-blocks").textContent = JSON.stringify(res.block, undefined, 2);
+    }
+    if (res.mempoolInfo) {
+      document.getElementById("result-mempool-info").textContent = JSON.stringify(res.mempoolInfo, undefined, 2);
+    }
+    if (res.transactions) {
+      document.getElementById("result-transactions").textContent = JSON.stringify(res.transactions, undefined, 2);
+    }
+    if (res["mempool-blocks"]) {
+      document.getElementById("result-mempool-blocks").textContent = JSON.stringify(res["mempool-blocks"], undefined, 2);
+    }
+  });
+  `,
+    esModule: `
+const { %{0}: { websocket } } = mempoolJS();
+
+const ws = websocket.initServer({
+options: ["blocks", "stats", "mempool-blocks", "live-2h-chart"],
+});
+
+ws.on("message", function incoming(data) {
+const res = JSON.parse(data.toString());
+if (res.block) {
+console.log(res.block);
+}
+if (res.mempoolInfo) {
+console.log(res.mempoolInfo);
+}
+if (res.transactions) {
+console.log(res.transactions);
+}
+if (res["mempool-blocks"]) {
+console.log(res["mempool-blocks"]);
+}
+});
+    `,
+  },
+  codeSampleMainnet: {
+    esModule: [],
+    commonJS: [],
+    curl: [],
+    response: ``
+  },
+  codeSampleTestnet: {
+    esModule: [],
+    commonJS: [],
+    curl: [],
+    response: ``
+  },
+  codeSampleSignet: {
+    esModule: [],
+    commonJS: [],
+    curl: [],
+    response: ``
+  },
+  codeSampleLiquid: {
+    esModule: [],
+    commonJS: [],
+    curl: [],
+    response: ``,
+  },
+  codeSampleBisq: {
+    esModule: [],
+    commonJS: [],
+    curl: [],
+    response: ''
+  },
```

Comment:
Is there any way to have default values automatically set when rendering instead of having those empty objects everywhere? That would cleanup lot of unnecessary code in this file.